third_party/txt/src/txt/asset_data_provider.cc

```diff
@@ -54,6 +58,7 @@ void AssetDataProvider::RegisterTypeface(sk_sp<SkTypeface> typeface) {
 
 void AssetDataProvider::RegisterTypeface(sk_sp<SkTypeface> typeface,
                                          std::string family_name_alias) {
+  family_names_.push_back(family_name_alias);
```

Comment:
The cases where the same family name is specified again and an empty family name are not handled.

Reply:
done